### DISCUSSION TITLE
Hydro calibration question

### DISCUSSION THREAD

**Discussion by glemieux** [NGEET/fates#1020] [Q&A] [answered]
Per @rgknox request I've broken this out into another discussion question as it diverged from the original discussion topic in #1015.
> I have another question.When the plant hydro turned on, the canopy transpiration is always low, I try to modify some parameters ,shuch as _fates_hydro_avuln_gs ,fates_hydro_p50_gs ,fates_hydro_kamx_node_... but no obvious effects.
  
> If I want to make the value of canopy transpiration larger,what should I do? 
Here is the output from the first year, but most of  value should be in 2~3 mm/day
![image](https://user-images.githubusercontent.com/97998383/231802034-e8d2cb4b-92e7-44db-8116-6dac9c076399.png)

> I really do not konw what happens inside the model. Any help would be greatly appreciated!

_Originally posted by @LianGong98 in https://github.com/NGEET/fates/discussions/1015#discussioncomment-5605827_

**Comment by rgknox** [comment, marked as answer]
My sense is that there is not a short answer to this, as calibration is a dedicated art and science. But there are some important levers here that it can't hurt to talk about.

My first question is, is the LAI fairly close to what is observed? If its not, then perhaps it would be useful to initialize the simulation with inventory using static-stand-structure mode (see namelist variables), or perform a satelllite phenology type simulation. In these modes, you can control and fix the leaf area, and thus work on parameters affecting photosynthesis and transpiration without the leaf area responding or complicating your calibration.

My next question is, is the GPP also off, or is it just the transpiration?

If GPP is also low, then perhaps increasing stomatal conductance and/ or conductances in the stem and roots might help increase both. Likewise, improving the plants access to soil water could also help here. Ways to do this is to increase total root mass, increase specific root length (fates parameter), or perhaps get the root profile deeper.

If the GPP is not low, then perhaps try increasing conductance, but also decreasing parameters that would decrease the interstitial leaf CO2 concentrations, like vcmax...? Now that I think of it, increasing root mass would potentially increase transpiration and decrease NPP since it would increase access to water (assuming water availability is not limited) and increase respiration.

Again, I'm spit-balling here and just mentioning some levers. But I think there are others who have put more thought into this exercise (with hydro), and might have better/more accurate suggestions (@JunyanDing , @xuchongang , @bchristo , @pnlfang , @walkeranthonyp @alistairrogers )
